node/src/main/java/ch/prevo/open/node/data/provider/MatchNotificationListener.java

```diff
@@ -0,0 +1,13 @@
+package ch.prevo.open.node.data.provider;
+
```

Comment:
Think this interface could belong next to the JobXxxxxProvider interfaces in `data-model`

Reply:
Yes, absolutely right. Fixed it.